run_all_tests.sh

```diff
@@ -1,12 +1,29 @@
 #!/usr/bin/env bash
-# Runs all tests.
+# Builds the project and runs all tests.
 #
 # A JVM will be selected by JAVA_HOME environment variable, or, if it is not set,
 # inferred from the java executable available on the path.
 
 # Fail immediately in case of errors and/or unset variables
 set -eu -o pipefail
 
-echo "Start running all EJB tests"
+# Import necessary environment variables (see the tests_profile header comment for details).
+source exonum-java-binding-parent/tests_profile
```

Comment:
Shouldn't we change `source tests_profile` in other places as well, like `pom.xml`?

Reply:
As far as I can see everything fine. `tests_profile` moved together with `pom.xml` to the exonum-java-binding-parent dir